verifier/src/report_body.rs

```diff
@@ -29,10 +29,15 @@ impl<'a> ReportBodyVerifier<'a> {
 }
 
 impl Verifier for ReportBodyVerifier<'_> {
-    type Error = VerificationError;
+    type Error = AndError<VerificationError, VerificationError>;
```

Comment:
I would like to put `AndError` in `VerificationError` as a variant, but an `AndError` is actually a tree like structure so it will blow up the size of `VerificationError` thus blowing up the leaf of the tree containted in `AndError`

Reply:
It could be `Box<(VerificationError, VerificationError)>` in which case it would only be the size of a pointer on the target architecture.

Reply:
As in: 
```rust
pub enum VerificationError {
    /// A general error.
    General,
    
    //  . . . 

    AndError(Box<(VerificationError, VerificationError)>),
}
```

Reply:
Sorry, I forgot a constraint, we want this to be no-alloc